automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCECustomizationSpec.scala

```diff
@@ -26,72 +26,6 @@ final class NotebookGCECustomizationSpec
   implicit val (ronAuthToken, ronAuthorization) = getAuthTokenAndAuthorization(Ron)
   implicit def ronToken: AuthToken = ronAuthToken.unsafeRunSync()
 
-  "NotebookGCECustomizationSpec" - {
-    // Using nbtranslate extension from here:
-    // https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/nbTranslate
-    "should install user specified notebook extensions" in { billingProject =>
-      val translateExtensionFile = ResourceFile("bucket-tests/translate_nbextension.tar.gz")
-      withResourceFileInBucket(billingProject, translateExtensionFile, "application/x-gzip") {
-        translateExtensionBucketPath =>
-          val extensionConfig = UserJupyterExtensionConfig(
-            nbExtensions = Map("translate" -> translateExtensionBucketPath.toUri)
-          )
-          withNewRuntime(
-            billingProject,
-            request =
-              LeonardoApiClient.defaultCreateRuntime2Request.copy(userJupyterExtensionConfig = Some(extensionConfig))
-          ) { runtime =>
-            withWebDriver { implicit driver =>
-              withNewNotebook(runtime, Python3) { notebookPage =>
-                // Check the extensions were installed
-                val nbExt = notebookPage.executeCell("! jupyter nbextension list")
-
-                nbExt.get should include("translate_nbextension/main  enabled")
-                // should be installed by default
-                nbExt.get should include("toc2/main  enabled")
-
-                val serverExt = notebookPage.executeCell("! jupyter serverextension list")
-                serverExt.get should include("jupyterlab  enabled")
-                // should be installed by default
-                serverExt.get should include("jupyter_nbextensions_configurator  enabled")
-
-                // Exercise the translate extensionfailure_screenshots/NotebookGCECustomizationSpec_18-34-37-017.png
-                notebookPage.translateMarkup("Yes") should include("Oui")
-              }
-            }
-          }
-      }
-    }
-
-    "should give cluster user-specified scopes" in { billingProject =>
```

Comment:
I removed this test despite not being covered elsewhere because it seems to be testing a feature of the google cloud, @rtitle  thoughts?

Reply:
I support this -- feels like it could be unit tested too (Leo reads the scopes from the API request and passes it to Google).